File: week10/week10-2.py
#week10-2
def gcd(a,b):
  if a==0: return b
  if b==0: return a
  return gcd(b, a%b)
a = int(input('請輸入a:'))
b = int(input('請輸入b:'))
print('最大公因數:', gcd(a,b) )

File: week10/week10-2.cpp
#include <stdio.h>
int gcd(int a, int b) {
    if(a==0) return b;
    if(b==0) return a;
    return gcd (b, a%b);
}
int main()
{
    printf("請輸入a b兩個數字: ");
    int a,b;
    scanf("%d%d",&a, &b);
    printf("最大公因數: %d\n",gcd(a ,b) );
}
